scripts/test

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env sh
+
+set -e
+projectRoot="$(a="/$0"; a=${a%/*}; a=${a:-.}; a=${a#/}/; cd "$a/.." || return; pwd)"
+
+command="$*"
+project="ffc-template-node"
+service="ffc-template-node"
```

Comment:
Both of these variables have the same value and given the usage in the script, I don't see any value why a single value wouldn't suffice.